scripts/trlite

```diff
@@ -306,6 +306,11 @@ if [ "$RUN" == "all" -o "$RUN" == "1" ]; then
     echo "var client = ocf.client;" >> $TMPFILE
     echo "var server = ocf.server;" >> $TMPFILE
     try_command "ocf" make $VERBOSE JS=$TMPFILE ROM=256
+
+    # PME test
+    MODULES=(pme)
+    write_modules_test $TMPFILE $MODULES
+    try_command "ocf" make $VERBOSE JS=$TMPFILE ROM=256
```

Comment:
This test shouldn't also be called "ocf". :)

But... add pme to the "net" test above instead, it will fit and then take less time by combining builds.